contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -2826,17 +2827,34 @@ sp_droplinkedsrvlogin_internal(PG_FUNCTION_ARGS)
 				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 				 errmsg("Only @locallogin = NULL is supported. Configuring remote server access specific to local login is not yet supported")));
 
+	remove_trailing_spaces(servername);
+
+	/* Check if servername is valid */
+	get_foreign_server_oid(servername, false);
```

Comment:
Question: This will throw error in case foreign server does not exist at CREATE time, have we verified if this is expected in T-SQL? If yes, then can we have a test case for it?